pai-management/k8sPaiLibrary/template/kubelet.sh.template

```diff
@@ -44,7 +44,6 @@ docker run \
   --pod-manifest-path=/etc/kubernetes/manifests \
   --allow-privileged=true \
   --logtostderr=true \
-  --image-gc-high-threshold=98 \
-  --image-gc-low-threshold=95 \
   --pod-infra-container-image {{ clusterconfig['dockerregistry'] }}/pause-amd64:3.0 \
+  --eviction-hard=memory.available<5%,nodefs.available<5%,imagefs.available<5%,nodefs.inodesFree<5%,imagefs.inodesFree<5% \
```

Comment:
do you know why the previous config doesn't work? can we add a comment to state which version of kubelet support the flag?

Reply:
Flag --image-gc-high-threshold has been deprecated, This parameter should be set via the config file Flag --image-gc-low-threshold has been deprecated, This parameter should be set via the config file

Reply:
Some kubelet Garbage Collection features in this doc will be replaced by kubelet eviction in the future.

REF: https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/